sonar-php-plugin/src/test/java/org/sonar/plugins/php/reports/psalm/PsalmSensorTest.java

```diff
@@ -113,32 +127,48 @@ public void raise_issue() throws IOException {
     assertThat(thirdTextRange.start().lineOffset()).isEqualTo(4);
     assertThat(thirdTextRange.end().line()).isEqualTo(5);
     assertThat(thirdTextRange.end().lineOffset()).isEqualTo(16);
-
-    assertNoErrorWarnDebugLogs(logTester);
   }
 
   @Test
-  public void raise_issue_file_has_unix_absolute_paths() throws IOException {
+  void raiseIssueFileHasUnixAbsolutePaths() throws IOException {
     List<ExternalIssue> externalIssues = executeSensorImporting("psalm-report-abs.json");
     assertThat(externalIssues).hasSize(3);
 
     assertNoErrorWarnDebugLogs(logTester);
   }
 
   @Test
-  public void raise_issue_file_has_windows_absolute_paths() throws IOException {
+  void raiseIssueFileHasWindowsAbsolutePaths() throws IOException {
     List<ExternalIssue> externalIssues = executeSensorImporting("psalm-report-abs_win.json");
     assertThat(externalIssues).hasSize(3);
 
     assertNoErrorWarnDebugLogs(logTester);
   }
 
   @Test
-  public void raise_issue_with_missing_fields() throws IOException {
+  void raiseIssueWithMissingFields() throws IOException {
     List<ExternalIssue> externalIssues = executeSensorImporting("psalm-report-with-missing-fields.json");
     assertThat(externalIssues).hasSize(5);
 
     ExternalIssue first = externalIssues.get(0);
+    verifyIssue1(first);
+
+    ExternalIssue second = externalIssues.get(1);
+    verifyIssue2(second);
+
+    ExternalIssue third = externalIssues.get(2);
+    verifyIssue3(third);
+
+    ExternalIssue fourth = externalIssues.get(3);
+    verifyIssue4(fourth);
+
+    ExternalIssue fifth = externalIssues.get(4);
+    verifyIssue5(fifth);
```

Comment:
Nitpick and opinion wise, I let you decide which form you prefer.
```suggestion
    verifyIssue1(externalIssues.get(0));
    verifyIssue2(externalIssues.get(1));
    verifyIssue3(externalIssues.get(2));
    verifyIssue4(externalIssues.get(3));
    verifyIssue5(externalIssues.get(4));
```